compiler/rustc_codegen_ssa/src/codegen_attrs.rs

```diff
@@ -779,6 +784,45 @@ fn check_link_name_xor_ordinal(
     }
 }
 
+struct LinterStateMixedExportNameAndNoMangle {
```

Comment:
Let's reduce a bit the length of this struct.
```suggestion
struct MixedExportNameAndNoMangleState {
```